--- homebank-3.8/configure.ac~	2008-03-31 13:41:04.000000000 +0200
+++ homebank-3.8/configure.ac	2008-07-30 19:50:19.511400588 +0200
@@ -45,7 +45,7 @@
     if test x$noofx != xtrue
     then
         AC_CHECK_LIB(ofx, ofx_set_status_cb, OFX_0_7="-DOFX_0_7")
-        LDFLAGS="${LDFLAGS} -lofx"
+        DEPS_LIBS="-lofx ${DEPS_LIBS}"
         CFLAGS="${CFLAGS} $OFX_0_7"
     else
         noofx=true
